src/test/java/org/sonarlint/intellij/ui/nodes/IssueNodeTest.java

```diff
@@ -20,53 +20,16 @@
 package org.sonarlint.intellij.ui.nodes;
 
 import com.intellij.psi.PsiFile;
-import com.intellij.ui.SimpleTextAttributes;
-import icons.SonarLintIcons;
-import java.io.IOException;
 import org.junit.Test;
 import org.sonarlint.intellij.issue.LiveIssue;
-import org.sonarlint.intellij.ui.tree.TreeCellRenderer;
 import org.sonarsource.sonarlint.core.client.api.common.analysis.Issue;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 public class IssueNodeTest {
   private IssueNode node;
-  private TreeCellRenderer renderer = mock(TreeCellRenderer.class);
```

Comment:
Why is it ok to drop these tests?

Reply:
It's not great but the methods in `TreeCellRenderer` are now final and we can't verify their invocation anymore.